### DISCUSSION TITLE
No Play/Sound after new install with Spotify

### DISCUSSION THREAD

**Discussion by pabera** [MiczFlor/RPi-Jukebox-RFID#1350] [Q&A]
I just installed a fresh version of the Phoniebox including Spotify. 

To start simple, I copied a few MP3 files the classic way and wanted to play it with the Phoniebox player. I do get a successful message in the UI that the it would play the music, but it doesn't. I verified that the soundcard works, it does!

So I checked the Iris UI to play some music from Spotify, which also didn't work. I received the following error in the UI:

```
Mopidy: Failed to add some tracks
```

So I searched the discussions a bit and found @s-martin constantly say, execute `journalctl -u mopidy` and post what it says.

The log does not say much, the only error I could found is this:

```
raspberrypi mopidy[4326]: ERROR    [MainThread] mopidy.internal.log Loading logging config '/etc/mopidy/logging.conf' failed. 'formatters'
[...]
raspberrypi mopidy[4326]: ERROR    [SpotifyEventLoop] spotify.session Spotify login error: <ErrorType.BAD_USERNAME_OR_PASSWORD: 6>
```

I guess I might have entered the wrong Spotify device password which is different from the login.

1. Could this potentially be the issue why the Classic Player does not work?
2. How can I update the Spotify device password after the installation?
3. What other logs can I check to dive deep into this issue?

**Comment by s-martin** [comment]
Please check

`/etc/mopidy/mopidy.conf` or

` ${HOME_DIR}/.config/mopidy.conf`

**Comment by pabera** [comment]
I have been able to resolve the problem regarding Iris not playing music. 

> The mopidy service uses `/etc/mopidy/mopidy.conf` where as the Iris UI uses `${HOME_DIR}/.config/mopidy.conf` as a source. My password was indeed inserted wrongly. Because Iris needs the Spotify credentials, everything worked there. But as soon as Iris relied on mopidy to play music, it failed because it uses a different config. Once updated both, the music started playing.

But a second problem persists: I am still unable to play music via the the Phoniebox UI. I have added MP3s the to the default `audiofolder` and those folders also appear in the UI. When I want to play them, the UI says that it would play but it does not happen actually.

Where is the place to check logs to potentially find out more about this problem?

**Reply by s-martin** [reply]
Does audio work in this general?

**Reply by pabera** [reply]
Yes it does! Iris plays music

But the Phoniebox software does not pick up the state.. Its Play button does not change to the "Pause" symbol for example, indicating that something would be running.
